resources/zib/ext-MultidisciplinaryTeamMeeting.PatientConsultationLabel.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-MultidisciplinaryTeamMeeting.PatientConsultationLabel" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-MultidisciplinaryTeamMeeting.PatientConsultationLabel" />
+  <name value="ExtMultidisciplinaryTeamMeetingPatientConsultationLabel" />
+  <title value="ext MultidisciplinaryTeamMeeting.PatientConsultationLabel" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The concept PatientConsultationLabel [zib MultidisciplinaryTeamMeeting v1.0(2020EN)](https://zibs.nl/wiki/MultidisciplinaryTeamMeeting-v1.0(2020EN)) cannot be properly mapped in the Encounter resource, hence a custom extension is used." />
```

Comment:
Has this mapping been removed from Encounter.type.text, because it might not semantically represent a type in every case (according to issue #43)? The two examples in the zib itself can be seen as a type in my opinion, but in general this concept might contain a string that doesn't represent one (e.g. 'MDO meneer Janssen').

Reply:
Mainly because a pattern is added to .type (see slice multidisciplinaryTeamMeetingCode), and an extra mapping to .type.text would not be correct next to that in my opinion.